### ISSUE TITLE
support for batch normalization

### ISSUE THREAD

**Issue by yunbin** [saturn-lab/androidAudioRecg#5]
Is there libandroid_tensorflow_inference_java.jar or libtensorflow_inference.so can support inference for a neural network model with batch normalization layer on Android?
